tests/rptest/tests/controller_recovery_test.py

```diff
@@ -27,6 +29,27 @@ def new_controller_elected():
             return curr and curr != prev
 
         wait_until(new_controller_elected,
-                   timeout_sec=30,
-                   backoff_sec=2,
+                   timeout_sec=20,
```

Comment:
How is this value chosen?

Reply:
I usually start with 30 seconds and if it seems clear that it doesn't need to be that long I'll lower it a bit